tests/integration/test_loki_push_api.py

```diff
@@ -72,8 +72,17 @@ async def test_rule_files_ingested_by_loki(ops_test):
     await ops_test.model.wait_for_idle(status="active", timeout=1000)
 
     # now, make sure rules are present
+    expected = [
+        ["HighPercentageError"],
+        ["HighThroughputLogStreams"],
+        ["HighThroughputLogStreams"],
+        ["HighThroughputLogStreams"],
+        ["http-credentials-leaked"],
+    ]
     response = await client.rules()
     assert (await client.rules()).items() > {}.items()
```

Comment:
Feel free to remove this one - it is now redundant.

Reply:
Remove what? Do you mean `assert (await client.rules()).items() > {}.items()`

Reply:
Yes, it is obviated by your addition.